lizmap/tooltip.py

```diff
@@ -211,7 +211,7 @@ def create_popup_node_item_from_form(
                     a += h
                     continue
                 # If it is root children, store html in the right list
-                if isinstance(n, QgsAttributeEditorField):
+                if isinstance(n, QgsAttributeEditorField) or (isinstance(n, QgsAttributeEditorElement) and n.type() == 6):
```

Comment:
```suggestion
                if isinstance(n, QgsAttributeEditorField) or (isinstance(n, QgsAttributeEditorElement) and n.type() == 6):
                # TODO QGIS_VERSION_INT 3.30.0 Change the integer with the QGIS enum `Qgis.AttributeEditorType.TextElement`
```

Doing comparaison for number is not "human readable", so I would like to change ASAP with the proper QGIS API.